src/tool.rs

```diff
@@ -39,6 +39,25 @@ pub struct Tool {
     pub(crate) has_internal_target_arg: bool,
 }
 
+impl From<windows_registry::Tool> for Tool {
+    fn from(tool: windows_registry::Tool) -> Self {
+        let mut cc_tool = Self::with_family(
+            tool.path().into(),
+            ToolFamily::Msvc {
+                clang_cl: tool.is_clang_cl(),
+            },
+        );
+
+        cc_tool.env = tool
+            .env()
+            .into_iter()
+            .map(|(k, v)| (k.clone(), v.clone()))
+            .collect();
+
+        cc_tool
+    }
+}
```

Comment:
If the plan is to bump `windows-registry` versions without bumping `cc-rs`, then to _actually_ be SemVer compatible, we can't have this impl.

Reply:
Thanks fixed